TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -9,23 +9,40 @@
 
 logger = logging.getLogger(__name__)
 
-AuthMethod = Enum("AuthMethod", ["BASIC", "SIGV4"])
+AuthMethod = Enum("AuthMethod", ["NO_AUTH", "BASIC_AUTH", "SIGV4"])
 HttpMethod = Enum("HttpMethod", ["GET", "POST", "PUT", "DELETE"])
 
 SCHEMA = {
     "endpoint": {"type": "string", "required": True},
     "allow_insecure": {"type": "boolean", "required": False},
-    "authorization": {
-        "type": "dict",
-        "required": False,
-        "schema": {
-            "type": {
-                "type": "string",
-                "required": True,
-                "allowed": [e.name.lower() for e in AuthMethod],
+    'authorization': {
+        'type': 'dict',
+        'required': True,
+        'schema': {
+            'type': {
+                'type': 'string',
+                'required': True,
+                'allowed': [e.name.lower() for e in AuthMethod]
```

Comment:
haha, battle of the code formatters (I have no preference)

Reply:
Let me at least update so there is consistent quote usage